class.jetpack.php

```diff
@@ -7280,6 +7285,38 @@ public static function is_rewind_enabled() {
 		return $rewind_enabled;
 	}
 
+	/**
+	 * Checks for whether Jetpack Scan is enabled.
+	 * Will return true if the state of Scan is anything except "unavailable".
+	 *
+	 * @return bool|int|mixed
+	 */
+	public static function is_scan_enabled() {
+		if ( ! self::is_active() ) {
+			return false;
+		}
+
```

Comment:
Is there any quick way to see if the site supports scan. Do we have the product or plan stores already?